modules/javafx.controls/src/main/java/javafx/scene/control/TableView.java

```diff
@@ -407,25 +466,9 @@
      * hand side columns reach minimum size, the user cannot increase the size of
      * resized column any more.
      */
-    public static final Callback<ResizeFeatures, Boolean> CONSTRAINED_RESIZE_POLICY = new Callback<ResizeFeatures, Boolean>() {
-
-        private boolean isFirstRun = true;
-
-        @Override public String toString() {
-            return "constrained-resize";
-        }
-
-        @Override public Boolean call(ResizeFeatures prop) {
-            TableView<?> table = prop.getTable();
-            List<? extends TableColumnBase<?,?>> visibleLeafColumns = table.getVisibleLeafColumns();
-            Boolean result = TableUtil.constrainedResize(prop,
-                                               isFirstRun,
-                                               table.contentWidth,
-                                               visibleLeafColumns);
-            isFirstRun = ! isFirstRun ? false : ! result;
-            return result;
-        }
-    };
+    @Deprecated(since="20")
```

Comment:
In addition to the `@Deprecated` annotation, can you add an `@deprecated` javadoc tag indicating that it is replaced by `CONSTRAINED_RESIZE_POLICY_FLEX_TAIL`?